packages/tg3/src/query.rs

```diff
@@ -0,0 +1,193 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use tg_bindings::TgradeMsg;
+
+use cosmwasm_std::{CosmosMsg, Decimal};
+use tg_utils::Expiration;
+
+use crate::msg::Vote;
+
+/// This defines the different ways tallies can happen.
+/// Every contract should support a subset of these, ideally all.
+///
+/// The total_weight used for calculating success as well as the weights of each
+/// individual voter used in tallying should be snapshotted at the beginning of
+/// the block at which the proposal starts (this is likely the responsibility of a
+/// correct cw4 implementation).
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum ThresholdResponse {
```

Comment:
Is this object never constructed in the code?
I cannot find anything making "total_points" that would be needed to construct this

Reply:
I am surprised - I knew it is in ProposalResponse which is returned by voting-contract, but it turns out, that... ProposalResponse is redefined in voting-contract! Without this one. Just will remove along with the whole ProposalResponse.